script/Arcueid/Day 2/02_00_ARC02_1.txt

```diff
@@ -291,7 +291,7 @@ Hisui bows and leaves the room.
 [sha:7e8a37682a31290f2b2f88e15ffdf1b32f797d08]{
 -- Page 20, Offset 2652.
 -- 一階ロビーに降りる。
-I head down to the first-floor lobby.
+I head down to the first floor lobby.
```

Comment:
Hyphenated not correct here as an adjective?

Reply:
idk, im second guessing myself :sacchingcopium:. i'll circle back around later